config/__init__.py

```diff
@@ -17,7 +17,7 @@
 
 import requests
 
-from config.environment import *  # noqa
+import config.environment as cfg
```

Comment:
With this change, I think the errors we're seeing are because now we're not importing all the values from environment.py into the app's config (which happens via `app.config.from_object('config')`). 

I'm not sure how we would pull this off without redefining all the environment vars here...

`BASE_DIR = cfg.BASE_DIR` 🤔 

I'm not sure we actually want that. I'm curious if @karlcow has some other suggestions (though he is out sick for a few weeks).

Reply:
One possibility would be to create a package for environment and split out 3 modules for staging, production and localhost, https://docs.python.org/3/tutorial/modules.html#importing-from-a-package, defining them inside `__all__`.

We would also want to move the other vars from environment.py to inside config/__init__.py (it seems like there's a few things in there that aren't really environment related).

But that seems like a lot of work for little benefit.

Reply:
yes the change is a bit bigger in architecture than I was planning for at the beginning of this issue and probably harder to achieve. The project is suffering from his past. 

Probably we should drop this issue #3113 and we could create a new issue with a scarier title 👺 like "Webcompat app configuration architecture overhaul".